app/Notifications/CheckinAssetNotification.php

```diff
@@ -162,6 +162,7 @@ public function toMail()
         $message = (new MailMessage)->markdown('notifications.markdown.checkin-asset',
             [
                 'item'          => $this->item,
+                'status'        => $this->item->assetstatus->name ?: '',
```

Comment:
You should be able to use it in these spots too, @Godmartinz - just for uniformity and simplification